dotnet/src/SemanticKernel.Abstractions/AI/FunctionChoiceBehaviors/FunctionChoice.cs

```diff
@@ -9,6 +9,7 @@ namespace Microsoft.SemanticKernel;
 /// Represents an AI model's decision-making strategy for calling functions, offering predefined choices: Auto, Required, and None.
 /// Auto allows the model to decide if and which functions to call, Required enforces calling one or more functions, and None prevents any function calls, generating only a user-facing message.
 /// </summary>
+[Experimental("SKEXP0010")]
```

Comment:
0010 is reserved for OpenAI only, but this is related to our abstractions, might consider using the `0001` identifier instead.

```suggestion
[Experimental("SKEXP0010")]
```

Reply:
Yep, it makes sence.